dataactvalidator/sqs_work_dispatcher.py

```diff
@@ -0,0 +1,706 @@
+import logging
+import inspect
+import json
+import os
+
+import psutil as ps
+import signal
+import sys
+import time
+import multiprocessing as mp
+
+from botocore.exceptions import ClientError, EndpointConnectionError, NoCredentialsError, NoRegionError
+
+from dataactcore.aws.sqsHandler import sqs_queue
+from dataactvalidator.validator_logging import log_job_message
+
+# Not a complete list of signals
+# Only the following are allowed to be used with signal() on Windows
```

Comment:
Done 
Removed them. It may have made the code more portable to other projects that also target Windows, but no need here. We run on Unix-like OSs and tell those who would run it on their own to use Docker.